### DISCUSSION TITLE
[error] vaultwarden unuseable after update on testing branch

### DISCUSSION THREAD

**Discussion by t3dium** [dani-garcia/vaultwarden#2826] [Q&A] [answered]
after updating vaultwarden's image on the testing branch, i'm unable to start the docker container since /usr/bin/dumb-init doesn't exist, (tried wget'ing the executable as well and placing it in /usr/bin/, but no luck)

it worked when i switched to the latest image in the normal stable branch, so only the testing branch is currently broken

**version with bug:** latest version on testing branch as of 14/10/2022
**version working:** latest stable image, and the previous image on testing branch
**installation method:** docker
**OS:** ubuntu server

**Comment by BlackDex** [comment, marked as answer]
Current stable and testing are identical so that's not possible.

Also, dumb-init has been removed for a while already, so maybe you have it configured somewhere in the `CMD` or `ENTRYPOINT`, or even a custom script?

**Reply by t3dium** [reply]
oh yeah it was in my entrypoint for some reason, i suspect portainer might've been at fault for that. testing branch works now when removing it

**Comment by redstormsju** [comment]
I just updated my Vaultwarden this morning using Portainer…and I got the dumb-init problem which did not allow me to start my container, so dumb-init still coming up for some.  My resolution was to delete the container and image…and i reinstalled it directly from the docker UI….and it worked.

**Reply by pianoquintet** [reply]
> I just updated my Vaultwarden this morning using Portainer…and I got the dumb-init problem which did not allow me to start my container, so dumb-init still coming up for some. My resolution was to delete the container and image…and i reinstalled it directly from the docker UI….and it worked.

I have the same problem.  I was able to get Vaultwarden to work again temporarily by recreating the container using the old image.

I would now like to try and upgrade again but I want to avoid having to delete the container as I fear I would lose all my env variables, settings, etc.  Any suggestions?

**Reply by BlackDex** [reply]
Adjusting the `CMD` and `ENTRYPOINT` values?

**Reply by pianoquintet** [reply]
Thanks, I set them back to default and it worked.

**Reply by BlackDex** [reply]
Not sure why portainer copies those values while they can change in between versions.

**Reply by major-mayer** [reply]
That's a good question, probably to save the transfer the values when a user changed them intentionally. 
I had exactly the same problem, and reverting the values to default worked.

**Reply by Sturmkater** [reply]
How did you set them back to default?

**Reply by nivek1385** [reply]
The entry for 1.25.2 works and that's the one with /usr/bin/dumb-init,-- as the entry point. 1.26.0 fails while spamming the log with "Command not found: /start.sh"
